lms/static/scripts/frontend_apps/components/StudentSelector.tsx

```diff
@@ -86,29 +77,21 @@ export default function StudentSelector<Student extends StudentOption>({
             title="Previous student"
             variant="dark"
           />
-          <Select
-            classes="min-w-[12rem] xl:w-[20rem]"
+          <SelectNext
+            classes="md:!w-[12rem] lg:!w-[16rem] xl:!w-[20rem]"
```

Comment:
The need for `!` here is unfortunate. Can you think of a way we could avoid that in future.

Reply:
I will check this again, because I think this was the result of a couple of tests I did, but prefixed classes should have precedence over the non-prefixed and built-in `w-full` one, so if I'm not wrong, it should work the same if these classes are set without the important modifier.

Reply:
Yep, I can confirm the `!` are not needed.